jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -90,7 +90,61 @@ typedef enum
   JERRYX_ARG_REQUIRED
 } jerryx_arg_optional_t;
 
+/**
+ * Indicates the rounding policy which will be choose to transform a integer.
+ */
+typedef enum __attribute__((packed))
```

Comment:
The need for an attribute signals that something is non-standard here. Why is this needed? I guess it is to ensure that jerryx_arg_int_option_t fits into extra_info. But that can be solved in other ways. BTW, is it necessary to expose jerryx_arg_int_option_t to the user? Why not have the jerryx_arg_(u)intX functions accept both a jerryx_arg_round_t and a jerryx_arg_clamp_t argument and let those functions encode (pack) the enums into extra_info? E.g., by casting the enum values to integers and shifting-bitmasking them together into a single int value. As a lib user, you could still create mappings easily but without a need for an extra (public) type, e.g.,

```c
jerryx_arg_int8 (&num0, 0, 0, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED)
jerryx_arg_int8 (&num0, JERRYX_ARG_ROUND, 0, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED)
jerryx_arg_int8 (&num0, 0, JERRYX_ARG_NO_CLAMP, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED)
jerryx_arg_int8 (&num0, JERRYX_ARG_ROUND, JERRYX_ARG_NO_CLAMP, JERRYX_ARG_COERCE, JERRYX_ARG_REQUIRED)
```